lib/service_check.pm

```diff
@@ -334,7 +334,7 @@ sub check_services {
         my $srv_proc_name = $service->{$s}->{srv_proc_name};
         my $support_ver = $service->{$s}->{support_ver};
         my $service_type = 'Systemd';
-        next unless (($service->{$s}->{before_migration} eq 'PASS') && _is_applicable($srv_pkg_name));
```

Comment:
suggest put comments here why why we remove this.

Reply:
OK, I add the reason to the commit and the description, thanks.

Reply:
We need add comments into the code, that's can make our life more easier.